### PR TITLE
docs: improve contribution section

### PR DESCRIPTION
**Pull request by escapedcat**
Draft to improve the contribution section. Happy for any feedback!

### CODE REVIEW

**Review by secondl1ght — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -201,13 +201,29 @@ Currently, there is one native companion app available to connect to Tor nodes:
...
+### Find a task
 
-If you're a developer looking to help but not sure where to begin, check out the issues that have specifically been marked as being friendly to new contributors [here](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+project%3AgetAlby%2Flightning-browser-extension%2F2).
+We use the [Development Project Board](https://github.com/orgs/getAlby/projects/10/views/2) to plan to-dos. Best choose something from the to-do column. (If there is nothing for you, feel free to pick something from the backlog)
 
-> Alby enforces [Conventional Commits Specification](https://www.conventionalcommits.org/en/) for adding human and machine readable meaning to commit messages, make sure to read and follow them.
+If you're a developer looking to help but not sure where to begin, check out the issues that have specifically been [marked as being friendly to new contributors](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+project%3AgetAlby%2Flightning-browser-extension%2F2).
```
Comment (bumi) [inline README.md:208]: should we only link to the "good first use" issues? 
https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

I guess the filter with the project board limits it too much?

and what can we give non developers?

If you're interested in the UI and design of Alby, check out the [issues marked as design](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22design%22+) 

what else can we add? documentation/guides?

Comment (escapedcat) [reply README.md]: Good point! Extended the section. Have a look.  
Also had a look at other projects, i.e. https://github.com/Microsoft/vscode#contributing  
I think there's space to split things up in separate doc-files or wiki-pages but we can do that when there's a need and this Readme is getting i.e. too big.
What do you think?

File: README.md [hunk 2]
```diff
@@ -201,13 +201,42 @@ Currently, there is one native companion app available to connect to Tor nodes:
...
+#### Developer
 
-If you're looking for a bigger challenge, before opening a pull request please [create an issue](https://github.com/bumi/lightning-browser-extension/issues/new) or [join our community chat](https://bitcoindesign.slack.com/archives/C02591ADXM2) to get feedback, discuss the best way to tackle the challenge, and to ensure that there's no duplication of work.
+- Check out the issues that have specifically been [marked as being friendly to new contributors](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3Adesign+label%3A%22good+first+issue%22)
+- You can also review open PRs
+
+#### Designer
+
+- Check out the issues that have specifically been [marked with "design"](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22design%22)
+- We also have a [Figma Design Guide Project](<https://www.figma.com/file/xEPJFGFdc3HD0JS1ax2IpH/Extension-(MVP)>) which you can have a look at
```
[the remark this answers is not in the document]
Comment (escapedcat) [reply README.md:216]: Happy to update the link. Is the general draft ok?

[the remark this answers is not in the document]
Comment (escapedcat) [reply README.md:216]: We can adjust the link later. If the general changes are ok, let's get this in first?

[the remark this answers is not in the document]
Comment (escapedcat) [reply README.md]: Got the link from @dvoroneca and updated it.

**Review by bumi — APPROVED** [review]

**Pull request merged**